examples/connext_dds/logging_config/c++/application.h

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
I think this file should be added to gitignore and not committed.

Reply:
I would like to remove it too. But this file is needed if not the compilation will fail.

This file is not generated as part of `connextdds_add_example`.